test/misc.test

```diff
@@ -174,6 +174,7 @@ exec_test_set_prereq jq
 exec_test_set_prereq openssl
 exec_test_set_prereq veritysetup
 test_expect_success base64,dd,jq,openssl,veritysetup "verity" "
+	mkdir -p input &&
```

Comment:
Please call `rm -rf input &&` before that, just like `rauc_setup` does, to make sure that we have a defined state.